CRM/Core/BAO/CustomField.php

```diff
@@ -1372,10 +1372,7 @@ public static function getFileURL($contactID, $cfID, $fileID = NULL, $absolute =
         ) {
           [$path] = CRM_Core_BAO_File::path($fileID);
           $fileHash = CRM_Core_BAO_File::generateFileHash(NULL, $fileID);
-          $url = CRM_Utils_System::url('civicrm/file',
-            "reset=1&id=$fileID&fcs=$fileHash",
-            $absolute, NULL, TRUE, TRUE
-          );
+          $url = CRM_Utils_System::url('civicrm/file', "reset=1&id=$fileID&fcs=$fileHash", $absolute);
```

Comment:
To make my suggestion more concretely, I think we should force frontEnd when `$absolute` is true:

```suggestion
          $url = CRM_Utils_System::url('civicrm/file', "reset=1&id=$fileID&fcs=$fileHash", $absolute, NULL, TRUE, $absolute);
```